src/gwt/src/org/rstudio/studio/client/workbench/views/environment/dataimport/DataImportOptionsUiCsvLocale.java

```diff
@@ -123,10 +131,46 @@ private void initializeDefaults()
       encoding_.addItem("UTF-8-MAC", "UTF-8-MAC");
       encoding_.addItem("UTF8", "UTF8");
       encoding_.addItem("UTF8-MAC", "UTF8-MAC");
```

Comment:
Some of these aren't very common. Should we use the list from `.rs.iconvcommon()` here?